src/open_inwoner/pdc/views.py

```diff
@@ -216,6 +216,7 @@ def get_context_data(self, **kwargs):
         if product.contacts.exists():
             anchors.append(("#contact", _("Contact")))
 
+        context["meta_description"] = product.summary
```

Comment:
@alextreme I'm not entirely sure if this is the right field to use as description in the meta tags, what do you think?

Reply:
Using the product summary is fine